src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.RemoteServer.cs

```diff
@@ -843,6 +869,7 @@ public async Task SendAsync_SendSameRequestMultipleTimesDirectlyOnHandler_Succes
         [OuterLoop("Uses external server")]
         [Theory]
         [MemberData(nameof(Http2Servers))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/53018", TestPlatforms.Browser)]
```

Comment:
Does `SendAsync_RequestVersion20_ResponseVersion20` work? Is there something particular about `SendAsync_RequestVersion20_ResponseVersion20IfHttp2Supported` that fails that is not a part of `SendAsync_RequestVersion20_ResponseVersion20`

Reply:
some of the HTTP/2 is probably broken, but I want to investigate that under #53018 
The `SendAsync_RequestVersion20_ResponseVersion20` is Windows only test